dockerclient.go

```diff
@@ -26,7 +27,7 @@ const (
 	// (v1.17) and
 	// ListVolumes, {Remove,Create}Volume, ListNetworks,
 	// {Inspect,Create,Connect,Disconnect,Remove}Network (v1.21)
-	APIVersion = "v1.15"
+	APIVersion = "v1.20"
```

Comment:
do you HAVE to bump the api ? It will have some side effects (swarm not being able to talk to engine with version < 1.20)